lib/mongo_client.js

```diff
@@ -434,10 +472,18 @@ MongoClient.prototype.startSession = function(options) {
  * @param {Function} operation An operation to execute with an implicitly created session. The signature of this MUST be `(session) => {}`
  * @return {Promise}
  */
+const withSessionSchema = {};
```

Comment:
This is a documentation problem: the options for `withSession` should be the same as the options for `startSession`

Reply:
`startSession` sets `explicit` to true [here](https://github.com/mongodb/node-mongodb-native/pull/1865/files/e2a9a57b416fe4a48725fcefe581b949844e2f40#diff-af7f90364fd34f588f43f92c13a92ab6L415), but `withSession` does not.